keyboards/wheatfield/fanoe_hotswap/config.h

```diff
@@ -0,0 +1,55 @@
+#ifndef CONFIG_H
+#define CONFIG_H
```

Comment:
Using `#pragma once` in place of traditional include guards is recommended. Remove these two lines, as well as the `#endif` at the bottom of the file, and put `#pragma once` here instead.

Reply:
OK fixed